templates/example_python/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# First, specify the base Docker image.
+# You can see the Docker images from Apify at https://hub.docker.com/r/apify/.
+# You can also use any other image from Docker Hub.
+FROM apify/actor-python:3.9
+
+# Second, copy just requirements.txt into the actor image,
+# since it should be the only file that affects "pip install" in the next step,
+# in order to speed up the build
+COPY requirements.txt ./
+
+# Install the packages specified in requirements.txt,
+# Print the installed Python version, pip version
+# and all installed packages with their versions for debugging
+RUN echo "Python version:" \
+ && python --version \
+ && echo "Pip version:" \
+ && pip --version \
+ && echo "Installing dependencies from requirements.txt:" \
+ && pip install -r requirements.txt \
+ && echo "All installed Python packages:" \
+ && pip --version
```

Comment:
I assume this should be something else than `pip --version` no?

Reply:
Yeah, good catch, forgot to update this line after copy paste.